src/uu/df/src/table.rs

```diff
@@ -410,6 +404,14 @@ impl Table {
             rows.push(total_row.get_values());
         }
 
+        for row in &rows {
```

Comment:
could you please a comment explaining what this block is doing thanks

Reply:
done :-)
I also added a unittest meanwhile